sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/CosmosResponseFactory.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.CosmosItemResponse;
+
+/**
+ * This response factory converts response messages
+ * to the corresponding type response using the
+ * CosmosClient serializer
+ */
+public interface CosmosResponseFactory {
+
+    <T> CosmosItemResponse<T> createItemResponse(CosmosItemResponse<byte[]> responseMessage, Class<T> classTypee);
```

Comment:
minor typo in `classTypee` -> `classType`

Reply:
haha. thanks for catching that